src/containers/Review/overview/Overview.tsx

```diff
@@ -29,12 +30,13 @@ export const Overview: React.FC<{
   const organisation = org?.name
   const { started, completed } = getDates(activityLog)
   const stage = current.stage.name
+  const { isMobile } = useViewport()
 
   return (
     <div id="overview">
       <Segment basic>
         <Message info icon>
-          <Icon name="info circle" color="teal" />
+          {!isMobile && <Icon name="info circle" color="teal" />}
           <Message.Content>
             <div className="flex-row-center wrap">
```

Comment:
![Screenshot 2023-11-07 at 12 16 18 PM](https://github.com/msupply-foundation/conforma-web-app/assets/5456533/d66bee03-77f5-4192-be72-ccebeb4b9e2d)

Just need to get the "Extend deadline" UI to wrap on mobile. (Hit me up in Telegram if you can't find an application with this set -- or just change the condition to `true` temporarily to make it show up.)

Reply:
Ohh, forgot this one...  Will get it